### ISSUE TITLE
Key Remapping for PageUp & PageDown

### ISSUE THREAD

**Issue by gungwald** [AppleWin/AppleWin#549]
This is an enhancement request. I'd like to be able to map my PageUp key to Alt-UpArrow so that I can hit PageUp in Merlin and have it work. Same thing for PageDown.

**Comment by sicklittlemonkey** [comment]
It might be better to do this outside of AppleWin.
https://autohotkey.com/boards/viewtopic.php?t=12276

Cheers,
Nick.

**Comment by tilleul** [comment]
I agree that the ability to remap a modern keyboard for specific Apple software might be interesting ... as well as remap some AppleWin shortcuts/features ...
